tests/sqlx/format.spec.ts

```diff
@@ -146,7 +146,56 @@ WHERE
   n < 8
 QUALIFY
   MOD(ROW_NUMBER() OVER (), 2) = 0
+`);
+    });
+
+    test("format triple quoted string", async () => {
+      expect(await formatFile(path.resolve("examples/formatter/definitions/triple_quoted.sqlx")))
+      .equal(`config {
+  type: "table"
+}
+
+SELECT
+  '''1''' AS single_line,
+  """multi
+  line
+    string
+      with indent""" AS multi_line,
+  REGEXP_CONTAINS("\\n  abc\\n  ", r'''
+abc
+''') AS multi_line_regex,
+  """
+This project is ...
+  "\${database()}"!!
+""" AS with_js
+
+post_operations {
+  select
+    """1""" as inner_sql
+}
 `);
     });
   });
+
+  suite("formatter todos", () => {
```

Comment:
I realized that previously, whether using single or triple quotes, template strings inside string literals could not be formatted. I made it like this because there was no way to represent todo tests.